src/main/java/parser/CustomerParser.java

```diff
@@ -0,0 +1,86 @@
+package parser;
+
+
+import customer.Customer;
+import exceptions.CustomerException;
+import exceptions.CliRentalException;
+
+
+public class CustomerParser {
+
+    private static final String ADD_CUSTOMER_COMMAND = "add-user";
+
+    /**
+     * Creates new customer object based on user input.
+     *
+     * @throws CustomerException if input is not compliant with format.
+     * @throws NumberFormatException if the age and contact content are not integer string.
+     */
+    public static Customer parseIntoCustomer(String userInput) throws CustomerException,
+            NumberFormatException {
+        userInput = userInput.substring(ADD_CUSTOMER_COMMAND.length()).trim();
+        String[] parameters = { "/u" , "/a" , "/c"};
+        String[] parameterContents;
+        if(isValidSequence(parameters, userInput)){
+            parameterContents = parseParameterContents(parameters, userInput);
+        } else{
+            throw CustomerException.addCustomerException();
+        }
+
+        String username = parameterContents[0];
+        int age = Integer.parseInt(parameterContents[1]);
+        int contactNumber = Integer.parseInt(parameterContents[2]);
+        return new Customer(username , age, contactNumber );
```

Comment:
Be mindful about the horizontal whitespace you use throughout the codebase.
```suggestion
        return new Customer (username , age, contactNumber);
```